core/arch/arm/arm.mk

```diff
@@ -26,6 +26,7 @@ CFG_MMAP_REGIONS ?= 13
 CFG_RESERVED_VASPACE_SIZE ?= (1024 * 1024 * 10)
 
 ifeq ($(CFG_ARM64_core),y)
+$(call force,CFG_WITH_LPAE,y)
```

Comment:
Wouldn't it make more sense to move this just below the `CFG_CORE_ARM64_PA_BITS` line?